spec/routines/newflow/update_user_from_sheerid_webhook_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+require 'vcr_helper'
+
+module Newflow
+  RSpec.describe UpdateUserFromSheeridWebhook, type: :routine, vcr: VCR_OPTS  do
+    context 'when success' do
+      subject(:run) { described_class.call(verification_id: verification_id) }
```

Comment:
suggestion to name the subject better here than run.   what is the object under test?  a subject would the sheer_id_webhook?  then, you'd do an action on the subject in the test.   

for example, take this test for Person.  the subject is the object under test (the Person).  sometimes you might not use one since it's implicit, but often an explicit subject is helpful.  

the subject is the (noun) object under test. 

```
describe Person do
  context "born 19 years ago" do
    subject { Person.new(:birthdate => 19.years.ago }
    it { should be_eligible_to_vote }
    its(:age) { should == 19 }
    it "should be younger than a 20 year old" do
      twenty_year_old = Person.new(:birthday => 20.years.ago)
      subject.should be_younger_than(twenty_year_old)
    end
  end
end
```

or 

```
describe "named subject" do
  subject(:a) { A.new }
  it "has been instantiated" do
    expect(a).to be_an(A)
  end
end
```

Reply:
Thank you for explaining that very well 😃 . I _did_ mean "run" as a noun, in this case. Like, "go for a run." I can't think of a better name for this SUT. I'm open to specific suggestions 😄 .